src/UI/Internal/Text.elm

```diff
@@ -365,3 +367,28 @@ combinedAttrs { ellipsis } =
 
     else
         []
+
+
+ellipsizedText : RenderConfig -> TextSize -> String -> Element msg
+ellipsizedText cfg size content =
+    let
+        lineHeightSize =
+            lineHeight (isMobile cfg) size
+    in
+    content
+        |> Html.text
+        |> Element.html
+        |> Element.el
+            (ellipsisAttrs lineHeightSize)
+
+
+ellipsisAttrs : Int -> List (Attribute msg)
+ellipsisAttrs lineHeightSize =
+    [ ( "text-overflow", "ellipsis" )
+    , ( "white-space", "nowrap" )
+    , ( "overflow", "hidden" )
+    , ( "display", "block" )
+    , ( "line-height", String.fromInt lineHeightSize ++ "px" )
```

Comment:
I had to also add this here, otherwise, the text will be cut for some TextSizes

Reply:
nice, I think it makes sense!